test/built-ins/Promise/prototype/finally/subclass-species-constructor-resolve-count.js

```diff
@@ -5,7 +5,6 @@ author: Jordan Harband
 description: finally on resolved Promise calls the SpeciesConstructor
 esid: sec-promise.prototype.finally
 features: [Promise.prototype.finally]
-flags: [async]
```

Comment:
Why is this incorrect? it depends on a resolution that doesn’t happen synchronously.

Reply:
We don't test the resolution here. We only check the result of calling `.finally` .. which happens synchronously, so there's no need for async.

Also, from https://github.com/tc39/test262/blob/master/CONTRIBUTING.md#flags:
**async** - defer interpretation of test results until after the invocation of the global $DONE function

Reply:
Both assertions in the test only perform checks on the returned object from `Promise.prototype.finally`, which doesn't require any async functionality. I don't know why there's an arrow function argument for `Promise.prototype.finally`, because it is essentially unused for this test. And the similar test https://github.com/tc39/test262/blob/master/test/built-ins/Promise/prototype/finally/subclass-species-constructor-reject-count.js doesn't have the async flag. And if the async flag is correct, the test is still incomplete, because async tests are required to call `$DONE`.